client/src/pages/Coinpage.js

```diff
@@ -0,0 +1,35 @@
+import Coins from '../components/Coins';
+import Headline from '../components/Headline';
+
+export default function Coinpage({
+  filteredCoins,
+  onToggleFavorite,
+  allCoins,
+}) {
+  return (
+    <>
+      <Headline Headlinetext="Market Cap." />
+      {filteredCoins.map((coin) => {
+        return <Coins coin={coin} onToggleFavorite={onToggleFavorite} />;
+      })}
+    </>
+  );
+}
+
+// const [filteredToken, setFilteredToken] = useState([]);
```

Comment:
Unused code shouldn't be here

Reply:
Forgot to delete it. Thank you for the great review!